### PR TITLE
CB-3: Add Cohort Builder Tabs

### PR DESCRIPTION
**Pull request by danielonodje**
Summary:
This PR add the tabs component which acts as a container for the cohort builder search elements.
JIRA Issue: https://issues.openmrs.org/browse/CB-3
Screenshot:
https://snag.gy/Lf9ISA.jpg

### CODE REVIEW

**Comment by dkayiwa** [comment]
@andela-odaniel can you squash the commits into one as advised at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by danielonodje** [comment]
@dkayiwa Commits squashed 😁

**Comment by dkayiwa** [comment]
@andela-odaniel can you style with the same effects as the openmrs reference application? The main area is white and only the left over sides are gray.
In other words, when one click the cohort builder app, they should still maintain a consistent look and feel of the main screen.

**Comment by danielonodje** [comment]
I've made the tabs to look like the reference application. Please note that the page is grey because there is no content. The content will have a white background. The breadcrumb and the tabs are the only elements of this task.

**Comment by dkayiwa** [comment]
Take a look at how an empty page should look like: http://qa-refapp.openmrs.org/openmrs/adminui/metadata/visits/visitattributetypes/manageVisitAttributeTypes.page#/list

As you can see, the content area has a background of white while the outside area is gray.

File: app/index.html [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-  </div>
-  <span class="app-title">CohortBuilder</span>
-</header>
-<div class="clear"></div>
-<div id="body-wrapper">
-  <div id="app">If you see this text, look for an error in your JavaScript Console.</div>
-</div>
 
+<body>
+    <div id="app">If you see this text, look for an error in your JavaScript Console.</div>
```
Comment (dkayiwa) [inline app/index.html:15]: We need to localise this text.

File: app/js/components/breadCrumb/breadCrumbComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    componentDidMount(){}
+
+    render(){
+        return (
+            <div className="breadcrumb">
+                <a href="/openmrs" className="breadcrumb-item">
+                    <span className="glyphicon glyphicon-home breadcrumb-item" aria-hidden="true"></span>
+                </a>
+                <span className="glyphicon glyphicon-chevron-right breadcrumb-item separator" aria-hidden="true"></span>
+                <span className="title breadcrumb-item">Cohort Builder</span>
```
Comment (dkayiwa) [inline app/js/components/breadCrumb/breadCrumbComponent.jsx:16]: We need to localise this text too.

**Comment by dkayiwa** [comment]
Generally, we need to localise text such that when users switch language, they do get the appropriate translations.

**Pull request merged**